_i18n/en/_posts/2017-09-14-ruby-on-rails-support.md

```diff
@@ -1,43 +1,43 @@
 ---
 layout: post
-title: Localize your Ruby on Rails app with GitLocalize
+title: Localize Your Ruby on Rails App with GitLocalize
 languages:
   - ja
 image: gitlocalize.png
 ---
 
-Are you struggling on managing i18n YAML files and its localization work with Ruby on Rails? GitLocalize is now here to help you all the tedious i18n work with Rails!
+Struggling to manage your I18n YAML files and the accompanying localization work with Ruby on Rails? Well, we have great news for you. GitLocalize is now here to help you take care of all that tedious Rails I18n work!
```

Comment:
I think `i` in `i18n` should be lowercase. https://www.w3.org/International/questions/qa-i18n

Reply:
Thanks for pointing this one out! I wasn't familiar with i18n, and a quick google search kept returning results for "I18n".  I've went ahead and changed them all back to "i18n"